src/main/webapp/app/exercises/modeling/manage/modeling-exercise-detail.component.html

```diff
@@ -16,7 +16,7 @@ <h2><span jhiTranslate="artemisApp.modelingExercise.detail.title">Modeling Exerc
                     <span jhiTranslate="artemisApp.textExercise.checkPlagiarism">Check Plagiarism</span>
                 </a>
             </div>
-            <div *ngIf="modelingExercise.releaseDate != undefined && moment(modelingExercise.releaseDate).isBefore(moment())" class="mt-3">
+            <div *ngIf="modelingExercise.releaseDate != undefined && dayjs(modelingExercise.releaseDate).isBefore(dayjs())" class="mt-3">
```

Comment:
I suggest again moving the evaluation to the component to not expose the date API to the template.

Reply:
I created a [confluence page ](https://confluence.ase.in.tum.de/display/ArTEMiS/Code+Review) for general implementation/style aspects which can be handled separately after discussion how to proceed with your suggestions.